idseq_dag/engine/pipeline_flow.py

```diff
@@ -206,6 +213,14 @@ def write_invalid_input_json(self, error_json):
 
         idseq_dag.util.s3.upload_with_retries(local_input_errors_file, self.output_dir_s3 + "/")
 
+    def create_status_json_file(self):
+        ''' Create [stage name]_status.json, which will include step-level job status updates to be used in idseq-web. '''
```

Comment:
nit: I know we don't follow this anywhere else in this repo, but PEP8 docstrings would be double-quotes (https://www.python.org/dev/peps/pep-0257/)